hack/static/versions.yaml

```diff
@@ -0,0 +1,473 @@
+  AppName: grafana-logging-6.28.0
+  AppVersion: 9.0.1
+  Chart: grafana
+  DisplayName: Grafana Logging
+  Enabled: false
+  LatestVersion: 6.31.1
+  Path: /services/grafana-logging
+  Repo: grafana.github.io
+  Type: platform
+  Version: 6.28.0
+- AppID: grafana-loki
+  AppName: grafana-loki-0.48.4
+  AppVersion: 2.5.0
+  Chart: loki-distributed
+  DisplayName: Grafana Loki
+  Enabled: false
+  LatestVersion: 0.49.0
+  Path: /services/grafana-loki
+  Repo: grafana.github.io
+  Type: platform
+  Version: 0.48.4
+- AppID: istio
+  AppName: istio-1.13.5
+  AppVersion: 1.14.1
+  Chart: istio
+  DisplayName: Istio Service Mesh
+  Enabled: false
+  LatestVersion: 1.14.1
+  Path: /services/istio
+  Repo: mesosphere.github.io-charts-staging
+  Type: catalog
+  Version: 1.13.5
+- AppID: jaeger
+  AppName: jaeger-2.29.0
+  AppVersion: 1.34.1
+  Chart: jaeger-operator
+  DisplayName: Jaeger
+  Enabled: false
+  LatestVersion: 2.32.2
+  Path: /services/jaeger
+  Repo: jaegertracing-github-io
+  Type: catalog
+  Version: 2.29.0
+- AppID: karma
+  AppName: karma-2.0.1
+  AppVersion: v0.70
+  Chart: karma
+  DisplayName: ""
+  Enabled: true
+  LatestVersion: 2.0.1
+  Path: /services/karma
+  Repo: mesosphere.github.io-charts-stable
+  Type: internal
+  Version: 2.0.1
+- AppID: karma-traefik
+  AppName: karma-traefik-0.0.1
+  AppVersion: 0.0.1
+  Chart: thanos-traefik
+  DisplayName: ""
+  Enabled: true
+  LatestVersion: 0.0.1
+  Path: /services/karma-traefik
+  Repo: mesosphere.github.io-charts-staging
+  Type: internal
+  Version: 0.0.1
+- AppID: kiali
+  AppName: kiali-1.49.0
+  AppVersion: v1.52.0
+  Chart: kiali-operator
+  DisplayName: Kiali
+  Enabled: false
+  LatestVersion: 1.52.0
+  Path: /services/kiali
+  Repo: kiali-org
+  Type: catalog
+  Version: 1.49.0
+- AppID: knative
+  AppName: knative-0.3.10
+  AppVersion: v0.22.3
+  Chart: knative
+  DisplayName: Knative
+  Enabled: false
+  LatestVersion: 0.3.10
+  Path: /services/knative
+  Repo: mesosphere.github.io-charts-staging
+  Type: catalog
+  Version: 0.3.10
+- AppID: kommander
+  AppName: kommander-v2.3.0-dev
+  AppVersion: v2.3.0-dev
+  Chart: kommander
+  DisplayName: ""
+  Enabled: false
+  LatestVersion: v2.3.0-dev
+  Path: /services/kommander
+  Repo: mesosphere.github.io-kommander-charts
+  Type: internal
+  Version: v2.3.0-dev
+- AppID: kommander-appmanagement
+  AppName: kommander-appmanagement-v2.3.0-dev
+  AppVersion: v2.3.0-dev
+  Chart: kommander-appmanagement
+  DisplayName: ""
+  Enabled: false
+  LatestVersion: v2.3.0-dev
+  Path: /services/kommander-appmanagement
+  Repo: mesosphere.github.io-kommander-charts
+  Type: internal
+  Version: v2.3.0-dev
+- AppID: kube-oidc-proxy
+  AppName: kube-oidc-proxy-0.3.1
+  AppVersion: v0.3.0
+  Chart: kube-oidc-proxy
+  DisplayName: kube-oidc-proxy
+  Enabled: false
+  LatestVersion: 0.3.1
+  Path: /services/kube-oidc-proxy
+  Repo: mesosphere.github.io-charts-staging
+  Type: internal
+  Version: 0.3.1
+- AppID: kube-prometheus-stack
```

Comment:
That comment is more accurate actually here for the kps app where we are deploying all the other components